tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -38,6 +38,35 @@ def test_format_number(number, expected, transformer):
     assert transformer().format_number(number) == expected
 
 
+@pytest.mark.parametrize(
+    "value, expected",
+    (
+        (123, "123%"),
+        ("123.4", "123.4%"),
+        ("123.40", "123.40%"),
+        ("1000", "1000%"),
+        (0, "0%"),
+        (0.00, "0%"),
+        ("", "%"),
+    ),
+)
+def test_format_percentage(value, expected, transformer):
+    assert transformer().format_percentage(value) == expected
+
+
+@pytest.mark.parametrize(
+    "unit, value, unit_length, expected",
+    (
+        ("centimeter", "123", "short", "123 cm"),
+        ("kilometer", "123", "long", "123 kilometre"),
+        ("mile", "123", "short", "123 mi"),
+        ("mile", "123", "narrow", "123mi"),
```

Comment:
Worth doing one without a unit length, to test the default.

Reply:
Done

Reply:
Still outstanding. Passing `None` in wont use our default. It uses babel's default `long`.
To use our default the logic need to to like `unit_length = unit_length or "short"`